contracts/LicenseToken.sol

```diff
@@ -102,6 +112,7 @@ contract LicenseToken is ILicenseToken, ERC721EnumerableUpgradeable, AccessManag
             uint256 tokenId = $.totalMintedTokens++;
```

Comment:
this could end up being quite gas intensive operation if user mints for example 1000 licenses.

Could `totalMintedTokens` be increased all at once by the amount?

Reply:
addressed.